aries_vcx_core/Cargo.toml

```diff
@@ -28,6 +30,7 @@ indy-vdr = { version = "0.3.4", default-features = false, features = ["ffi", "lo
 # - update libvdrtools to use =0.3.6 ursa
 indy-credx = { git = "https://github.com/anonyome/indy-shared-rs.git", rev = "7342bc624d23ece8845d1a701cd2cdc9cd401bb0", optional = true }
 libvdrtools = { path = "../libvdrtools", optional = true }
+indy-api-types = { path = "../libvdrtools/indy-api-types" }
```

Comment:
As discussed privately; it's not ideal IMO that this is a forced dependencies, as IMO we really want to be moving away from anything vdrtools related, or rather avoid modular_libs consumers having to include any vdrtools deps. 

I see now that you're importing this here such that you can update `errors/mod.rs` from:
```
#[cfg(any(feature = "vdrtools", feature = "modular_libs", feature = "vdr_proxy_ledger"))]
mod mapping_vdrtools;
```
to
```
mod mapping_indy_api_types;
```

Where `mapping_indy_api_types` is necessary for non-vdrtools traits (indy_vdr_ledger), since indy_vdr_ledger relies on IndyError from indy-api-types.

Reply:
Anyway, i know you already know all this, I'm just catching up on it... 

this import hasn't made it worse, just highlighted to me that the modular_libs feature is leaking some vdrtools stuff

Reply:
perhaps as a note, you could add a comment explaining that this is temporarily whilst we are de-vdrtooling the codebase

Reply:
Thanks George, yes, we are on the same page here~ 

I'll omit the note as addressing the problem you highlighted should be relatively imminent and simple right after https://github.com/hyperledger/aries-vcx/pull/914 is in.